render-release-notes.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+
+me=$(basename $0)
+
+function showUsage() {
+	cat<<-EOF
+Usage: $me --installer <msi|inno|nuget>
+   or: $me [--output <dir> [--css <path>]]
```

Comment:
Well, when we remove the `-i` switch, this becomes a non-issue, because it totally makes sense to be able to run the script and specify the CSS location without necessarily changing the output location of the Release Notes file.